.github/workflows/actionlint.yml

```diff
@@ -0,0 +1,13 @@
+
+name: Lint GitHub Actions Workflows
+on:
+  push:
+  pull_request:
```

Comment:
Possibly consider limiting runs of this workflow only to when things it cares about change?

```
    paths:
      - .github/**
```